script.js

```diff
@@ -1,4 +1,69 @@
+var currentGameMode = "waiting for user name";
+var userName = "";
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  var myOutputValue = "";
+  var computerChosenSecretWord = chooseSecretWord();
+  // if the user didn't enter a name
+  if (isNaN(Number(input)) == true) {
+    myOutputValue = "sorry please enter a name";
+  } else {
+    //otherwise play the game
+    userName = input;
+    currentGameMode = "scissors paper stone game";
+    myOutputValue = `Hello ${input}!`;
+  }
+  if (currentGameMode == "scissors paper stone game") {
+    // code for random computer word
+    // if the input is outside scissors, paper or stone
+    if (input != "scissors" || input != "paper" || input != "stone") {
+      return `sorry please key in either scissors, paper or stone;`;
+    }
+    // if win
+  }
+  if (
+    (input == "scissors" && computerChosenSecretWord == "paper") ||
+    (input == "paper" && computerChosenSecretWord == "stone") ||
+    (input == "stone" && computerChosenSecretWord == "scissors")
+  ) {
+    myOutputValue = `win`;
+  }
+  // if lose
+  if (
+    (input == "scissors" && computerChosenSecretWord == "stone") ||
+    (input == "paper" && computerChosenSecretWord == "scissors") ||
+    (input == "stone" && computerChosenSecretWord == "paper")
+  ) {
+    myOutputValue = `lose`;
+  }
+  // if draw
+  if (
+    (input == "scissors" && computerChosenSecretWord == "scissors") ||
+    (input == "paper" && computerChosenSecretWord == "paper") ||
+    (input == "stone" && computerChosenSecretWord == "stone")
+  ) {
+    myOutputValue = `draw`;
```

Comment:
When checking for win, lose or draw, it should be in if, else if, else statements. This is because only one conditional statement should happen. If you use multiple if statements, it sends the message that more than one if statement could run